models/src/main/scala/coop/rchain/models/rholangn/RhoType.scala

```diff
@@ -33,20 +33,21 @@ sealed trait RhoTypeN {
 trait AuxParN extends RhoTypeN
 
 /** Rholang element that can be processed in parallel, together with other elements */
-sealed trait ParN extends RhoTypeN {
+sealed trait ParN extends RhoTypeN
 
-  /** Combine two pars for their parallel execution */
-  def combine(that: ParN): ParN = combinePars(this, that)
-}
 object ParN {
   def fromBytes(bytes: Array[Byte]): ParN = parFromBytes(bytes)
+  def toBytes(p: ParN): Array[Byte]       = parToBytes(p)
```

Comment:
Here you have an object so toBytes is a static function. There is some justification behind putting an alias on a trait to have a dot notation (though it should not be done), but why do this alias? Instead of `ParN.toBytes(p)` just do `parToBytes(p)`. The same for `combine`.